packages/wallet/lib/getApiUrl.ts

```diff
@@ -0,0 +1,7 @@
+export function getApiUrl(): string {
+  if (process.env.NODE_ENV === 'production') {
+    return 'https://byor-l2beat.fly.dev'
```

Comment:
store this string also in env

Reply:
I think this will introduce some friction in local testing of production build of the wallet, if the API address starts changing a lot I'll remember to move it to the env